src/org/zaproxy/zap/extension/websocket/files/scripts/templates/websocketpassive/Passive WebSocket Scan Template.js

```diff
@@ -0,0 +1,81 @@
+// Passive scan rules should not send messages
+// Right click the script in the Scripts tree and select "enable"  or "disable"
+
+var PluginPassiveScanner = Java.type("org.zaproxy.zap.extension.websocket.pscan.WebSocketPluginPassiveScanner");
+
+OPCODE_CONTINUATION = 0x0;
+
+OPCODE_TEXT = 0x1;
+OPCODE_BINARY = 0x2;
+
+OPCODE_CLOSE = 0x8;
+OPCODE_PING = 0x9;
+OPCODE_PONG = 0xA;
+
+RISK_INFO 	= 0;
+RISK_LOW 	= 1;
+RISK_MEDIUM = 2;
+RISK_HIGH 	= 3;
+
+CONFIDENCE_LOW = 1;
+CONFIDENCE_MEDIUM = 2;
+CONFIDENCE_HIGH = 3;
+
+
+/**
+ * This function is Optional. By default allows all the message to pass at the scan phase.
+ * Return {@code true} to pass WebSocket message to scan function.
+ * @param msg WebSocket message (WebSocketMessage DTO)
+ */
+function applyScanToMessage(msg) {
+    if(msg.opcode == OPCODE_TEXT && !msg.isOutgoing){
```

Comment:
This equality seems not working fine. Any ideas why?

Reply:
It's working here.

Reply:
@ManosMagnus Is this still an issue or can the two conversations be resolved?

Reply:
Still remaining an issue for me